pomodoro/src/main/java/pomodoro/dto/PomodoroDTO.java

```diff
@@ -0,0 +1,34 @@
+package pomodoro.dto;
+
+public class PomodoroDTO {
+
+    private int pomID;
+    private String pName;
```

Comment:
If you introduce the https://projectlombok.org/, you will save yourself of writing quite a lot of boilerplate code. In order to use it, you also need to install a plugin for IntellIj/Eclipse.